src/test/java/hmwk16.java

```diff
@@ -0,0 +1,55 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+
+public class hmwk16 {
+    @Test
+    public static void newPlaylist() throws InterruptedException {
+
+        ChromeOptions chrome_Profile = new ChromeOptions();
+        chrome_Profile.addArguments("--disable-notifications");
+
+       //login
+        WebDriver driver = new ChromeDriver(chrome_Profile);
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+        String url = "https://bbb.testpro.io/#!/home";
+        driver.get(url);
+
+        WebElement emailField = driver.findElement(By.cssSelector("[type= 'email']"));
+        emailField.click();
+        emailField.sendKeys("seamugg@yahoo.com");
+
+        WebElement passwordField = driver.findElement(By.cssSelector("[type = 'password']"));
+        passwordField.click();
+        passwordField.sendKeys("te$t$tudent");
+
+        WebElement submitButton = driver.findElement(By.cssSelector("[type = 'submit']"));
+        submitButton.click();
+
+        //create new playlist with xpath locators
+
+
+        WebElement xPlaylist = driver.findElement(By.xpath("//*[@title= 'Create a new playlist']"));
+        xPlaylist.click();
```

Comment:
I recommend add the "Thread.sleep(5000);" after the web element has loaded, and then you can add a click method

For example

```
WebElement xPlaylist = driver.findElement(By.xpath("//*[@title= 'Create a new playlist']"));
Thread.sleep(5000);
 xPlaylist.click();
```